packages/app/src/components/Dimensions/DataDimension/Item.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { colors } from '../../../colors';
+import { RemoveSelectedItemButton } from './buttons';
+
+const styles = {
```

Comment:
We are moving to a convention of having the style objects in separate files inside a styles/ folder. Check elsewhere in the repo to get an idea.